cypress/e2e/SubdomainNavigation.cy.ts

```diff
@@ -0,0 +1,221 @@
+import { NFTPageSelector } from '../../src/pages/NFT/selectors';
+import { DomainsPageSelector } from '../../src/pages/Domains/selectors';
+import { SubdomainViewSelector } from '../../src/features/view-subdomains/selectors';
+
+describe('Subdomain Navigation', () => {
+	const rootDomainZna = 'wilder';
+	const subdomainZna = `${rootDomainZna}.snowflake`;
+
+	it('successfully loads correct page for the root domain', () => {
+		cy.visitRootDomain();
+
+		// assert url
+		cy.assertUrl('contain', rootDomainZna);
+		cy.assertUrl('not.contain', `${rootDomainZna}.`);
+
+		const expectedHref = `/0.${rootDomainZna}/nfts`;
+		const expectedText = 'wilder';
+
+		// assert address bar anchor segment
+		cy.get('.main__header')
+			.find('a')
+			.should('contain', `${rootDomainZna}`)
+			.should('have.length', 1)
+			.and('have.attr', 'href', expectedHref)
+			.and('have.text', expectedText);
+
+		// assert expected elements
+		cy.assertElementIsVisible(DomainsPageSelector.DOMAINS_PAGE_CONTAINER);
+		cy.assertElementIsVisible(SubdomainViewSelector.VIEW_SUBDOMAINS_SECTION);
+	});
+
+	it('successfully loads directly to a given domain', () => {
+		cy.visit(`/0.${subdomainZna}/nfts`);
+
+		// assert url
+		cy.assertUrl('contain', subdomainZna);
+
+		// assert address bar anchor segment
+		cy.get('.main__header')
+			.find('a')
+			.should('have.length', 2)
+			.each((a, index) => {
+				// assert anchor text for each anchor
+				const expectedText = index === 0 ? 'wilder' : 'snowflake';
+
+				cy.wrap(a).should('have.text', expectedText);
+			});
+	});
+
+	it('successfully loads to a given domain from root domain', () => {
+		cy.visitRootDomain();
+
+		// assert subdomain exists
+		cy.findByText(`0://${subdomainZna}`).should('exist');
+
+		cy.visit(`//0.${subdomainZna}/nfts`);
+
+		// assert url
+		cy.assertUrl('contain', subdomainZna);
+		cy.assertUrl('not.contain', rootDomainZna);
+
+		// assert address bar anchor segments
+		cy.get('.main__header')
+			.find('a')
+			.should('have.length', 2)
+			.each((a, index) => {
+				// assert href attribute and anchor text for each anchor
+				const expectedHref =
+					index === 0 ? `/0.${rootDomainZna}/nfts` : `/0.${subdomainZna}/nfts`;
+				const expectedText = index === 0 ? 'wilder' : 'snowflake';
+
+				cy.wrap(a)
+					.should('have.attr', 'href', expectedHref)
+					.should('have.text', expectedText);
+			});
+
+		// assert expected elements
+		cy.assertElementIsVisible(
+			SubdomainViewSelector.VIEW_SUBDOMAINS_BANNER_CONTAINER,
+		);
+		cy.assertElementIsVisible(
+			SubdomainViewSelector.VIEW_SUBDOMAINS_DETAILS_CARD_CONTAINER,
+		).and('contain', 'Winter Time');
+	});
+
+	it('can view grid view and go to corret domain', () => {
```

Comment:
nit: `can view grid view and navigate to clicked card's subdomain`
typo: `corret domain` -> `correct domain` (same w next test)

Reply:
nice spot, thanks!